types/index.d.ts

```diff
@@ -261,7 +415,8 @@ declare namespace postgres {
     | Helper<any>
     | Parameter<any>
     | ArrayParameter
-    | SerializableParameter[];
+    | Record<string, any> // implicit JSON
```

Comment:
I noticed new tests with an interesting name, I guess it's a new feature?

https://github.com/porsager/postgres/blob/43d78d548694dea7643b98c434f139ca4c18fdbf/tests/index.js#L113-L121

Reply:
Ah yes.. It's because we're now using the ParameterDescription from postgres to decide on types :)